kubernetes_state/auto_conf.yaml

```diff
@@ -2,6 +2,10 @@ docker_images:
   - kube-state-metrics
 
 init_config:
+  # Tags are reported as set by kube-state-metrics. If you want to translate
+  # them to other tags, you can use the labels_mapper dictionary
+  #  labels_mapper:
+  #    namespace: kube_namespace
```

Comment:
was planning to do costly processing of the dict on import, but went without at the end. Moving to instance